lib/api.rb

```diff
@@ -470,12 +695,22 @@ def validation_error(msg)
       ""
     end
   end
+
+  ### Organizations
+
+  post "/organizations" do
+    d= device_from_bearer
+	if !d
+	 return validation_error("invalid bearer")
+	end
+
+	return validation_error("Organizations not implemented yet")
+  end
```

Comment:
Is this needed if it doesn't do anything yet?

Reply:
Just to "remember", or if someone want to implement it.